rspec-core.gemspec

```diff
@@ -44,7 +44,7 @@ Gem::Specification.new do |s|
     s.add_runtime_dependency "rspec-support", "~> #{RSpec::Core::Version::STRING.split('.')[0..1].concat(['3']).join('.')}"
   end
 
-  s.add_development_dependency "cucumber", "~> 1.3"
+  s.add_development_dependency "cucumber", "~> 3.2.0"
```

Comment:
I believe we can go straight to 5.2.0 with a small change - replace `~@tag` with `not @tag` syntax in `cucumber.yml` and a couple of other files.

Reply:
Going straight to 5.2.0 would be ideal, but it doesn't support Ruby 2.3.